sources/packages/web/src/components/layouts/BodyHeaderContainer.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <!-- Header slot (with and without v-card) -->
+    <v-card v-if="card"><slot name="header"></slot></v-card>
+    <div v-else><slot name="header"></slot></div>
+
+    <!-- Default slot (with and without v-card) -->
+    <v-card v-if="card" class="mt-n1"><slot></slot></v-card>
+    <div v-else class="mt-n1"><slot></slot></div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  props: {
+    card: {
```

Comment:
can we give a proper name something like `isCard` or `enableCardView` or maybe something appropriate

Reply:
Done